library/ecdsa.c

```diff
@@ -301,7 +314,6 @@ static int ecdsa_signature_to_asn1( const mbedtls_mpi *r, const mbedtls_mpi *s,
 
     return( 0 );
 }
-
```

Comment:
Same as the other comment. THis is only a style change and not directly related to the PR